bash/deploy.sh

```diff
@@ -22,10 +23,18 @@ cd "${CLONE}"
 echo -e "\nBuilding Jekyll site:"
 rm -rf _site
 
-if [ -r ${DEPLOY_CONFIG} ]; then
-  jekyll build --config _config.yml,${DEPLOY_CONFIG}
+if [ -r ${BUNDLE} ]; then
+  if [ -r ${DEPLOY_CONFIG} ]; then
```

Comment:
@dmydlarz looks like code duplication for me. How about passing `$5` as `bundle exec` and just using it here.